webapp/src/main/webapp/js/officeonline.js

```diff
@@ -55,7 +60,7 @@
       } else {
         log("Cannot find .noPreview element");
       }
-    } else if ($elem.find("a.editOnlineBtn").length == 0) {
+    } else if ($elem.find("a.officeOnlineEditBtn").length == 0) {
```

Comment:
Proposal to use a CSS class name `editInOfficeOnline`

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/16/commits/9d563ac61939060cf045812a30a8980e0cd9b7e4